src/appsettings/settings.py

```diff
@@ -469,8 +469,10 @@ def raw_value(self):
             AttributeError: if the variable is missing.
             KeyError: if the item is missing from nested setting.
         """
-        if self.parent_setting is not None:
+        if isinstance(self.parent_setting, NestedDictSetting):
             return self.parent_setting.raw_value[self.full_name]
+        elif isinstance(self.parent_setting, NestedListSetting):
+            return self.parent_setting.raw_value[self.nested_list_index]
```

Comment:
`nested_list_index` is not defined.

Reply:
I was thinking about this. I want `nested_list_index` to only be defined when its parent setting is `NestedListSetting`. And I don't want to set it to 0, because that might cover up some errors in case I forgot to set it up. I could set it to `None` as default, if you think it's better.

Reply:
+1 for `None`

Reply:
Fixed.